.gitignore

```diff
@@ -8,6 +8,8 @@ bazel-bin
 bazel-out
 bazel-testlogs
 
+# Ignore bazelrc produced by `configure`
+.tf_configure.bazelrc
```

Comment:
`tools/python_bin_path.sh` is also generated by `./confiugure`. We should add it to `.gitignore` too.

Reply:
Ooo good catch, thanks for pointing this out. I'll update